.github/workflows/julia-exercise-ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        julia-version: ['1.0', '1.1', '1.2', '1.3', nightly]
+        julia-version: ['1.0', '1.1', '1.2', '1.3', '1.4', nightly]
```

Comment:
This might belong to a separate PR but since the exercise claims it's been tested on >= 1.0, I figured I'd add it here anyway.